### PR TITLE
gui: add button to open 21st Capital priority support page in browser

### LINKED ISSUE #3: [21st][GUI] - Add link to support page
There should be a button with a link to priority support, probably located in the top right as follows:

image

The colour of the button should be DAF7A6.

### PR DESCRIPTION
**Pull request by jp1ac4**
This is to resolve #3.

The button is in the top right corner of the Home page.

The button's background colour is as provided in the issue. In addition, I used #0F4539, also from the website, for the background when hovering over the button.

### CODE REVIEW

**Comment by jp1ac4** [comment]
Here's how the button looks:

image

When hovering over the button with the cursor:

image

**Comment by edouardparis** [comment]
I got:
  2024-09-26T15:05:53.305336Z ERROR liana_gui::app:305: Error opening 'https://21stcapital.com/priority-support/': No such file or directory (os error 2)

**Comment by jp1ac4** [comment]
The link may give a 404 currently, but it should hopefully open in the browser.

I've just pushed a change to modify the URL slightly and use only a single button appearance for now.

**Comment by edouardparis** [comment]
I successfully open the url, I had to install xdg-utils to make it work.

File: gui/src/app/mod.rs [hunk 1]
```diff
@@ -300,6 +300,12 @@ impl App {
                 )
             }
             Message::View(view::Message::Menu(menu)) => self.set_current_panel(menu),
+            Message::View(view::Message::OpenUrl(url)) => {
+                if let Err(e) = open::that(&url) {
```
Comment (edouardparis) [inline gui/src/app/mod.rs:304]: halloy is using that_detached https://docs.rs/open/latest/open/fn.that_detached.html
It may be better to use it instead.

Comment (jp1ac4) [reply gui/src/app/mod.rs:304]: Yep that seems better thanks, will change it. Would be good to test it also on Windows.

**Review by edouardparis — APPROVED** [review]
ACK c3d01d280c613031f9de4696bb5a27ed5b8190dc

**Pull request merged**